src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -4,7 +4,7 @@ import { entropyToMnemonic } from "bip39";
 import { Definitions, DeterministicSeedPhrase } from "@ganache/options";
 
 const unseededRng = seedrandom();
-
+const MAX_ACCOUNTS = 1000;
```

Comment:
I expect we'll tune this at some point but for now it was a quick/easy issue

Reply:
I think we should put this number much much higher, like 100000 accounts, maybe? We aren't so concerned with start up time here (users should notice extreme start-up performance issues) as we are with memory.

We want to switch to a lazy database model, where initial writes are to memory only, and only when memory starts getting full due to start writing to disk. Currently we just always write to disk. Once we switch to this lazy DB model the more memory we have available after start up, the better!

It looks like memory scales rather linearly, after initial startup garbage collection. Here is a graph of accounts (x axis) and memory (y axis) on my machine: https://www.wolframalpha.com/input?i=plot+%7C+%7B%7B10%2C+81%7D%2C+%7B100%2C+82%7D%2C+%7B1000%2C+88%7D%2C+%7B10000%2C+118%7D%2C+%7B25000%2C+181%7D%2C+%7B50000%2C300%7D%2C+%7B100000%2C+515%7D%7D

So what ever limit we choose is is arbitrary. /shrug

Reply:
:+1: https://github.com/trufflesuite/ganache/pull/3361/files#diff-b465f04541f0c5e2ad842f50433df5af81c1bdd98b7921c558dbd30d635ca25dR7